elastic/kibana
# [Rollup] Migrate to new ES client

## Summary

See https://github.com/elastic/kibana/issues/73973 

## How to test

1. Start Kibana and ES
2. Navigate to the rollup job sub section in the management section
3. Create a rollup using the wizard (make sure that pattern validation check is working correctly)
4. Delete a rollup job
